nilearn/regions/parcellations.py

```diff
@@ -5,17 +5,81 @@
 
 import numpy as np
 from joblib import Memory, Parallel, delayed
+from scipy.sparse import coo_matrix
 from sklearn.base import clone
 from sklearn.feature_extraction import image
 
-from nilearn.maskers import NiftiLabelsMasker
+from nilearn.maskers import NiftiLabelsMasker, SurfaceLabelsMasker
+from nilearn.maskers.surface_labels_masker import _signals_to_surf_img_labels
```

Comment:
```suggestion
from nilearn.maskers.surface_labels_masker import signals_to_surf_img_labels
```

if you are going to import in another 'subpackage' then it is not a function only private to the module it is in, so you should remove the leading `_`

Reply:
you will have to rename it in nilearn.maskers.surface_labels_masker